.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
+omit = **/migrations/*,**/settings/*,**/wsgi.py,**/tests/*,**/manage.py
+source = .
```

Comment:
Coverage only looks at files that are at the root of the specified directory or in a module (with `__init__.py`) and much of Archivematica is set up as a collection of scripts and isn't seen by coverage.  I think this will be more accurate:

```
source =
    src/dashboard/src/
    src/archivematicaCommon/lib/
    src/MCPClient/lib/
    src/MCPClient/lib/clientScripts/
    src/MCPServer/lib/
```

Sadly this brings test coverage down to 25%